home.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+
+    <!-- custom css -->
+    <link rel="stylesheet" href="style.css">
+
+    <!-- fontawesome cdn -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- google font -->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap');
+    </style>
+</head>
+
+<body>
+    <header class="toolbar">
+        <h1>Welcome</h1>
+        <i class="fa-solid fa-bars"></i>
+    </header>
+
+    <main class="headline">
+        <div class="primary">
+            <p>Hey there. I'm Mariane
+            </p>
+            <p id="green">I'm a software developer</p>
+        </div>
+        <div class="supporting-text">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+        </div>
+        <div class="social-media-icon">
+            <img src="photo/twitter.svg" alt="">
+            <img src="photo/Linkedin icon.svg" alt="">
+            <img src="photo/medium.svg" alt="">
+            <img src="photo/github.svg" alt="">
+            <img src="photo/angellist.svg" alt="">
+        </div>
```

Comment:
- I believe it will be better for the list of icons to use a `ul` tag to group them, and `li` per icon.
- Each icon should have a link that will help connect to your social media in an anchor tag (`a` tag).  Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Why don't we refactor your code to look something like this 👇🏽 
```
        <ul class="social-media-icon">
            <li><a href="http://yourtwitterhandle"><img src="photo/twitter.svg" alt=""></a></li>
            <li><a href="http://yourlinkedInhandle"><img src="photo/Linkedin icon.svg" alt=""></a></li>
            <li><a href="http://yourmediumhandle"><img src="photo/medium.svg" alt=""></a></li>
            <li><a href="http://yourgithubhandle"><img src="photo/github.svg" alt=""></a></li>
            <li><a href="http://yourangellisthandle"><img src="photo/angellist.svg" alt=""></a></li>
        </ul>
```

What do you think about this?? 💭

Reply:
I have upgraded my project thanks to your suggestions. So can I re-submit now? 🤓